sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSources/AzureApplicationInsightsDataFeedSource.cs

```diff
@@ -31,8 +31,6 @@ public AzureApplicationInsightsDataFeedSource(string applicationId, string apiKe
             Argument.AssertNotNullOrEmpty(apiKey, nameof(apiKey));
             Argument.AssertNotNullOrEmpty(query, nameof(query));
 
-            Parameter = new AzureApplicationInsightsParameter(azureCloud, applicationId, apiKey, query);
```

Comment:
As explained in the other comment, we're not creating the parameter during construction anymore. This is true for every data feed source.